src/main/java/org/sefglobal/scholarx/model/Profile.java

```diff
@@ -17,6 +17,9 @@
 @JsonIgnoreProperties({"createdAt", "updatedAt", "enrolledUsers"})
 public class Profile extends BaseScholarxModel {
 
+    public Profile() {
+    }
```

Comment:
If there are no other constructers Java provide no-args contractor by default. No need to code it

Reply:
Ah right